acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/transaction/perm/AccountSmartContractPermissioningAllowAccountTransaction.java

```diff
@@ -62,9 +60,11 @@ public Hash execute(final NodeRequests node) {
   }
 
   private String signedTransactionData() {
-    final BytesValue payload =
-        BytesValues.concatenate(
-            ADD_ACCOUNT_SIGNATURE, BytesValue.fromHexString("0x000000000000000000000000"), account);
+    final Bytes payload =
+        Bytes.concatenate(
+            ADD_ACCOUNT_SIGNATURE,
+            Bytes.fromHexString("0x000000000000000000000000"),
+            account.toBytes());
```

Comment:
I thought with the type extension we would not need to call `toBytes' on all the account objects?